### ISSUE TITLE
Sails 0.10.1 cannot find module grunt

### ISSUE THREAD

**Issue by gkaimakas** [balderdashy/sails#2059]
Just installed sails@0.10.1 and when I lift my server I get the following error:

```
error: Grunt :: module.js:340
    throw err;
          ^
Error: Cannot find module '/home/gkaimakas/node_modules/sails/node_modules/grunt-cli/bin/grunt'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:906:3
```

I saw similar problems in previous versions and every solution said to delete some backslashes from pathToSails.

```
var pathToSails = __dirname.replace(' ', '\ ') + '/../..';
```

the thing is that I cannot locate pathToSails anywhere. My server lifts fine though. Can anyone help ?

**Comment by Bazze** [comment]
This is absolutely still an issue. I'm experiencing the same as @rmarku.

**Comment by kt3k** [comment]
This seems fixed in [this commit](https://github.com/balderdashy/sails/commit/df0cafbb7a041bbf6f5e0e8babb8bec2ed89ac8a) and it's included in v0.11 or later.

**Comment by vigneshmack** [comment]
I created one notepad file I uses the command node C:\Users\Vignesh Ravi\AppData\Roaming\npm\node_modules\sails\bin\sails.js lift and save as sails.bat file and put into C:\Windows\System32.but still I got an error can anyone solve this error? Thanks in advance....
C:\Users\Vignesh Ravi\AppData\Roaming\npm>cd xyz

C:\Users\Vignesh Ravi\AppData\Roaming\npm\xyz>sails lift

C:\Users\Vignesh Ravi\AppData\Roaming\npm\xyz>node C:\Users\Vignesh Ravi\AppData\Roaming\npm\node_modules\sails\bin\sails.js lift
module.js:487
    throw err;
    ^

Error: Cannot find module 'C:\Users\Vignesh'
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.Module._load (module.js:437:25)
    at Function.Module.runMain (module.js:605:10)
    at startup (bootstrap_node.js:158:16)
    at bootstrap_node.js:575:3
